docs/js/search.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+    function displaySearchResults(results, store) {
+      var searchResults = document.getElementById('search-results');

```

Comment:
It's recommended to use let/const over var since they were introduced in js, as they are block-scoped as opposed to function-scoped, and therefore a bit less error-prone. More details here https://blog.usejournal.com/awesome-javascript-no-more-var-working-title-999428999994.

I'd change any var that doesn't change to 'const', and any that do to 'let'.